docs/static/advanced-pipeline.asciidoc

```diff
@@ -4,14 +4,72 @@
 In <<first-event>>, you created a basic Logstash pipeline to test your Logstash setup. In the real world, a Logstash
 pipeline is a bit more complex: it typically has one or more input, filter, and output plugins.  
 
-In this section, you create a Logstash pipeline that takes Apache web logs as input, parses those
+In this section, you create a Logstash pipeline that uses Filebeat to take Apache web logs as input, parses those
 logs to create specific, named fields from the logs, and writes the parsed data to an Elasticsearch cluster. Rather than
 defining the pipeline configuration at the command line, you'll define the pipeline in a config file. 
 
-The following text represents the skeleton of a configuration pipeline:
+To get started, go https://download.elastic.co/demos/logstash/gettingstarted/logstash-tutorial.log.gz[here] to
+download the sample data set used in this example. Unpack the file.
+
+
+[[configuring-filebeat]]
+==== Configuring Filebeat to Send Log Lines to Logstash
+
+Before you create the Logstash pipeline, you'll configure Filebeat to send log lines to Logstash.  https://github.com/elastic/beats/tree/master/filebeat[Filebeat] is a lightweight, resource-friendly tool that
+collects logs from files on the server and forwards these logs to your Logstash instance for processing. Filebeat is 
+designed for reliability and low latency. Filebeat uses the computing resources of the machine hosting the source data,
```

Comment:
> Filebeat uses the computing resources of the machine hosting the source data

This kind of indicates that FB uses a lot of resource on host machines. How about rephrasing this like "Filebeat is designed to ship logs with a light resource footprint on the host machine"